parm/config/config.fv3

```diff
@@ -59,6 +65,11 @@ case $case_in in
         export layout_y_gfs=4
         export nth_fv3=1
         export nth_fv3_gfs=1
+        #S4 is throttled to improve cycle runtime
+        if [[ $machine = "S4" && "${CASE_ENKF:-N/A}" == "C96" ]]; then
```

Comment:
Still don't like this carve-out for a machine few are likely to use. @aerorahul?

Reply:
I agree with @WalterKolczynski-NOAA . @DavidHuber-NOAA I'm guessing a 6x4 layout for C96 gdasfcst fail on S4? Is 6x4 for gfsfcst ok?

Reply:
Similar questions about the other resolution S4 if-blocks.

Reply:
@KateFriedman-NOAA A 6x4 layout for C96 runs just fine on S4 individually, but makes for a significantly slower cycle time.  Each user is restricted to 40 nodes at a given time.  A 4x4 job uses 4 nodes while a 6x4 uses 6 (10 jobs at a time vs 6) and the gain in runtime does not come close to making up for the extra time required to run 10 jobs 4 time vs 6 jobs 7 times.

Reply:
IMO, the resource allocation needs to be thought over again.  We have machines ranging from S4 to WCOSS2 which have significant differences in the node/core footprint.
I don't want to hold this PR up, but I would suggest until we get that piece resolved, we may have to sacrifice efficiency on S4. 
Happy to hear your thoughts.

Reply:
@aerorahul I can understand that and I can pull these changes.  However, I would like to note that for C384/C192 experiments, it is a significant cycle time difference (~2.25 hours/cycle) and does take up a lot of extra resources.

That said, perhaps a compromise would be to ask users on the wiki to contact me to help optimize their experiments when running on S4.  I can then keep these configuration files on S4 for reference.

Reply:
I've removed most of the optimizations from config.fv3 and config.resources.  I left just the resource modifications necessary for the analysis jobs (gdasanal, gfsanal, and gdaseobs).  This was discussed by email with @CoryMartin-NOAA and [summarized here for Jet](https://github.com/NOAA-EMC/global-workflow/issues/357#issuecomment-909483710).  The basics of that conversation are that if the number of cores allocated differs from the number of cores needed by these jobs, then some data may not be assimilated and will result in apparent spatial data gaps.  This is a result of S4 having 32 cores/node vs Hera and Orion's 40.

Below is a plot of the ATMS data points when using `npe_eobs=40` and `npe_node_eobs=16`.

<img width="1336" alt="Screen Shot 2021-05-27 at 9 22 30 AM" src="https://user-images.githubusercontent.com/69919478/192533277-a71eb4d4-1d8c-4151-899f-7996eca38465.png">

And here is the same plot when using `npe_eobs=40` and `npe_node_eobs=10`:

<img width="1328" alt="Screen Shot 2021-05-27 at 2 59 29 PM" src="https://user-images.githubusercontent.com/69919478/192533958-3a31d105-8911-4521-bc73-f6f5f3f291ec.png">